### PR TITLE
add onMtuChanged to BLEServerCallbacks

### PR DESCRIPTION
**Pull request by davidlehrian**
Add method onMtuChanged to the BLEServerCallbacks so the application can be notified if the MTU size changes.

### CODE REVIEW

**Review by chegewara — APPROVED** [review]

File: libraries/BLE/src/BLEServer.h [hunk 1]
```diff
@@ -134,6 +134,16 @@ class BLEServerCallbacks {
...
+
+	/**
+	 * @brief Handle a new client connection.
+	 *
+	 * When the MTU changes this method is invoked.
+	 *
+	 * @param [in] pServer A reference to the %BLE server that received the client connection.
+	 * @param [in] param A reference to esp_ble_gatts_cb_param_t.
+	 */
+	virtual void onMtuChanged(BLEServer* pServer, esp_ble_gatts_cb_param_t* param)
```
[the remark this answers is not in the document]
Comment (davidlehrian) [reply libraries/BLE/src/BLEServer.h]: Yeah I saw that.  I corrected the code and committed it to my fork.  Let me know if I need to do anything more.

[the remark this answers is not in the document]
Comment (chegewara) [reply libraries/BLE/src/BLEServer.h]: Odd, i commented this 2 days ago.

**Comment by davidlehrian** [comment]
Hmmm, just got an email this morning of the failure.

File: libraries/BLE/src/BLEServer.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidlehrian) [reply libraries/BLE/src/BLEServer.h]: Hmmm, just got an email this morning of the failure.

**Pull request merged**